stash_engine/app/models/stash_engine/identifier.rb

```diff
@@ -124,15 +120,6 @@ def target
       @target = tenant.full_url(StashEngine::Engine.routes.url_helpers.show_path(to_s))
     end
 
-    def create_or_get_identifier_state
-      return identifier_state unless identifier_state.nil?
-      c_a = CurationActivity.create(status: 'Unsubmitted', stash_identifier: self)
-      c_a.save!
-      @identifier_state = IdentifierState.create_identifier_state(self, c_a)
-      reload
-      @identifier_state
-    end
-
     # the search words is a special MySQL search field that concatenates the following fields required to be searched over
```

Comment:
removed all references to `curation_activities` (those are now associated with resources) and removed `identifier_state` references